includes/login-register.php

```diff
@@ -136,11 +136,19 @@ function edd_log_user_in( $user_id, $user_login, $user_pass, $remember = false )
 		'remember'      => $remember,
 	);
 
-	$user     = wp_signon( $credentials );
-	$redirect = wp_get_referer();
-	if ( empty( $redirect ) ) {
-		$redirect = edd_get_checkout_uri();
+	$user = wp_signon( $credentials );
+	$url  = wp_get_referer();
+	if ( empty( $url ) ) {
+		$url = edd_get_checkout_uri();
 	}
+	$redirect = add_query_arg(
+		array(
+			'checkemail'         => 'confirm',
+			'edd_reset_password' => 'confirm',
+			'edd_redirect'       => $url,
+		),
+		'wp-login.php'
```

Comment:
We shouldn't hard-code this in case someone has changed it (common with some security plugins).

```suggestion
		wp_login_url()
```